nyc_taxis/challenges/default.json

```diff
@@ -145,7 +178,23 @@
         {
           "name": "refresh-after-index",
           "operation": "refresh"
-        }
+        },
+        {
+          "operation": {
+            "operation-type": "force-merge",
+            "request-timeout": 7200{%- if force_merge_max_num_segments is defined %},
+            "max-num-segments": {{ force_merge_max_num_segments | tojson }}
+            {%- endif %}
+          }
+        },
+        {
+          "name": "refresh-after-force-merge",
+          "operation": "refresh",
+        },
+        {
+          "operation": "wait-until-merges-finish",
+          "include-in-reporting": true
```

Comment:
`include-in-reporting` is an operation parameter, not a task parameter so it needs to be defined on level deeper. This property also affects only the command line report; metrics are always pushed to the metrics store.